mithril-client-cli/src/main.rs

```diff
@@ -165,7 +165,7 @@ impl Source for Args {
 enum ArtifactCommands {
     /// Deprecated, use `cardano-db` instead
     #[clap(subcommand)]
-    #[deprecated(since = "0.7.2", note = "use `CardanoDb` commands instead")]
+    #[deprecated(since = "0.7.3", note = "use `CardanoDb` commands instead")]
```

Comment:
```suggestion
    #[deprecated(since = "0.7.3", note = "use `cardano-db` commands instead")]
```

Reply:
This is only in the code is not shown to users, so for me it make more sense to use the enum variant name than the its resulting command line value _(honestly this line is quite superfluous, we put in order to test how rust handles depreciation)_.